geonode/people/api/views.py

```diff
@@ -190,12 +193,7 @@ def transfer_resources(self, request, pk=None):
             == len(transfer_resource_subset)
         ):
 
-            if target_user == "DEFAULT":
```

Comment:
In the [original issue](https://github.com/GeoNode/geonode/issues/11995), to have a DEFAULT user was requested:

> The target user is set inside the payload. The “DEFAULT” constant string can be used to transfer ownership to the default user, which is the principal administrator.

Is a wanted changes now?